hack/cluster-deploy.sh

```diff
@@ -76,6 +76,6 @@ until _kubectl -n kubevirt get kv kubevirt; do
 done
 
 # wait until KubeVirt is ready
-_kubectl wait -n kubevirt kv kubevirt --for condition=Ready --timeout 180s || (echo "KubeVirt not ready in time" && exit 1)
+_kubectl wait -n kubevirt kv kubevirt --for condition=Ready --timeout 12m || (echo "KubeVirt not ready in time" && exit 1)
```

Comment:
12 minutes?

Reply:
Saw it miss the previous deadline a couple of times, but could even have been for the disks image provider loopback issue. Will revert it to the previous value to see if it's happening again